9c7718cddb229150791d8f8d13961979/update/sys_db_object_d47a3b43473d35104fb86285d36d4380.xml

```diff
@@ -9,30 +9,27 @@
         <create_access>true</create_access>
         <create_access_controls>true</create_access_controls>
         <delete_access>false</delete_access>
-        <extension_model/>
         <is_extendable>false</is_extendable>
         <label>API Point</label>
         <live_feed_enabled>false</live_feed_enabled>
         <name>x_snc_pointsthing_api_point</name>
-        <number_ref/>
+        <number_ref display_value="API">3ac05ae08305121059f46b70deaad3bc</number_ref>
         <provider_class/>
         <read_access>true</read_access>
         <scriptable_table>false</scriptable_table>
         <super_class/>
-        <sys_class_code/>
         <sys_class_name>sys_db_object</sys_class_name>
-        <sys_class_path/>
         <sys_created_by>earl.duque</sys_created_by>
         <sys_created_on>2022-10-13 03:05:34</sys_created_on>
-        <sys_id>d4ba90c1db629150791d8f8d139619cb</sys_id>
-        <sys_mod_count>0</sys_mod_count>
+        <sys_id>d47a3b43473d35104fb86285d36d4380</sys_id>
```

Comment:
Please clarify why the sys_id is changing. This will be a fundamentally different record which will drop the records from this table as the table itself drops on an install. Additionally, any references will break